slides/common/sampleapp.md

```diff
@@ -124,7 +124,7 @@ def hash_bytes(data):
 ```
 
 (Full source code available [here](
-https://github.com/jpetazzo/container.training/blob/8279a3bce9398f7c1a53bdd95187c53eda4e6435/dockercoins/worker/worker.py#L17
+https://@@GITREPO@@/blob/8279a3bce9398f7c1a53bdd95187c53eda4e6435/dockercoins/worker/worker.py#L17
```

Comment:
I didn't notice before that this link is to such a specific sha - is that for a reason?

Reply:
Yes! I'm pointing to a specific line number, so I want to make sure that it remains consistent even if the file is modified. (For instance, if a comment or even a blank line is added/removed, I want the `#L17` to still point to the right place.)

Reply:
Makes sense to me! Thanks.